cypress/integration/locks.js

```diff
@@ -1,8 +1,46 @@
 describe("Locks", function() {
-    // beforeEach(function() {
-    //     cy.getUserAEurBalance().as("startingAeurBalance");
-    //     cy.getUserEthBalance().as("startingEthBalance");
-    // });
+    const getLock = (prodId, disbursedAmount) => {
+        cy.get("[data-testid=lockMenuLink]").click();
 
-    it("Should lock and release A-EUR");
+        cy.get(`[data-testid=selectLockProduct-${prodId}]`).then(() => {
+            cy
+                .get("[data-testid=lockAmountInput]")
+                .type(disbursedAmount.toString())
+                .should("have.value", disbursedAmount.toString())
+        });
+        cy.get(`[data-testid=selectLockProduct-${prodId}]`).click();
+
+        cy.get(`[data-testid=submitButton]`).click();
+        cy.get("[data-testid=EthSubmissionSuccessPanel]").contains("New Lock submitted")
+        cy.get("[data-testid=EthSubmissionSuccessPanel] >[data-testid=msgPanelOkButton]").click();
+        
+        return cy.get("[data-testid=EthConfirmationReceivedPanel]").within(() => {
+            cy.contains("New lock");
+            cy.contains("Tx hash:");
+        });
+    };
+    
+    beforeEach(function() {
+        cy.getUserAEurBalance().as("startingAeurBalance");
+        cy.getUserEthBalance().as("startingEthBalance");
+    });
+    
+    it("Should lock A-EUR", function() {
+        getLock(8, 50).then(() => {
+            cy.assertUserAEurBalanceOnUI(this.startingAeurBalance - 50);
+            cy.get("[data-testid=EthConfirmationReceivedPanel] > [data-testid=msgPanelOkButton]").click();
+        });
+    });
+
+    it("Should release locked A-EUR", function() {        
+        getLock(8, 50).then(() => {
+            cy.get("[data-testid=EthConfirmationReceivedPanel]").within(() => {
+                cy.contains("12. confirmation", { timeout: 20000 });
```

Comment:
don't wait for 12th confirmation, it's not necessary on local (any confirmation after the receipt is just simulated on local ganache anyways). just assert "1. New lock"  within `EthConfirmationReceivedPanel`